<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowContourGroupDataItem_PropertyDialog</class>
 <widget class="QDialog" name="Post3dWindowContourGroupDataItem_PropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contour Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="rangeTab">
      <attribute name="title">
       <string>Faces</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="Post3dWindowFaceListSettingEditWidget" name="faceListWidget" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>464</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="colormapTab">
      <attribute name="title">
       <string>Colormap</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <widget class="ColorMapSettingEditWidget" name="colorMapWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="importButton">
           <property name="text">
            <string>&amp;Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>&amp;Export...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorMapSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/scalarstocolors/colormapsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Post3dWindowFaceListSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>post/post3d/post3dwindowfacelistsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Post3dWindowContourGroupDataItem_PropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Post3dWindowContourGroupDataItem_PropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
